src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class ColorSupplier {
+}
```

Comment:
The class `ColorSupplier` should have a method to generate a random color. Currently, the class is empty. Consider implementing a method that returns a random color value, possibly by using an array of color names or an enum representing different colors.